cli/src/types.ts

```diff
@@ -61,7 +62,7 @@ export type SelectedComponents =
 
 export type AvailablePackages = {
   name: (typeof availablePackages)[number];
-  type: 'navigation' | 'styling' | 'authentication' | 'internationalization';
+  type: 'navigation' | 'styling' | 'authentication' | 'internationalization' | 'analytics';
```

Comment:
@dannyhw We'll want to add this into #325

Reply:
ok no problem